### ISSUE TITLE
[BUG] Incorrect order for EventPriority

### ISSUE THREAD

**Issue by pinkuth** [WaterdogPE/WaterdogPE#325]
Event priorities are currently prioritised as `LOWEST -> HIGHEST` due to looping `EventPriority.values()` with the current order of the enum.

**To Reproduce**
1. Create two events for the same event, one with `EventPriority.LOWEST` and other `EventPriority.NORMAL` (default)
2. The event with `EventPriority.LOWEST` will be called first, not the other.

**Expected behavior**
The events with the highest priority should be called first meaning the order should be `HIGHEST -> LOWEST` not `LOWEST -> HIGHEST` (as it currently is).

**Screenshots**
- https://i.imgur.com/be3LHU8.png (Code)
- https://i.imgur.com/rY0hPZt.png (Console)

**Platform Information:**
 - Game Platform: Windows 10
 - Game (Client) Version: 1.21.50.7
 - Server OS: Ubuntu 24.04
 - Java Version: Java 17
 - Server Deployment Method: Docker

**Comment by DavyCraft648** [comment]
Higher priority events are intended to have the final say in how an event is handled so they are executed last

**Comment by pinkuth** [comment]
> Higher priority events are intended to have the final say in how an event is handled so they are executed last

If that's the case then why do the docs for the `EventPriority` enum state "HIGHEST is called first" and "LOWEST is called last"?

If that is indeed the intended way the priority for events is meant to work, then the docs should be updated to reflect the actual order.

**Comment by TobiasGrether** [comment]
Yeah docs were wrong, sorry! I corrected them accordingly. Besides, I recommend posting code samples as actual code instead of screenshots.
